pkg/util/k8sutil/backup.go

```diff
@@ -92,7 +92,9 @@ func CreateAndWaitPVC(kubecli kubernetes.Interface, clusterName, ns, pvProvision
 		return err
 	}
 
-	err = retryutil.Retry(4*time.Second, 5, func() (bool, error) {
+	// TODO: We were having unpredictable long time on GKE. See https://github.com/kubernetes/kubernetes/issues/40972 .
```

Comment:
We set timeout to 60s here since PVC binding could take up to 60s for GCE/PD. See ...